### PR TITLE
Add new product-led metrics to handbook

### PR DESCRIPTION
**Pull request by jyavorska**
This is intended to implement the agreed upon proposals from https://docs.google.com/document/d/1o0dLmdRRI6uWIuAg_8VQw25KnTM1CBDKKR2K91SxpAI/edit?skip_itp2_check=true&pli=1# into the handbook.

At the time of creation there were a few points left to align on, but getting into a PR can help us make the proposal and discussion more concrete.

### CODE REVIEW

File: handbook/ops/bizops/user_definitions.md
Comment (jyavorska) [inline handbook/ops/bizops/user_definitions.md:40]: This was an open discussion item from the source document that we should complete.

File: handbook/ops/bizops/user_definitions.md
Comment (jyavorska) [inline handbook/ops/bizops/user_definitions.md:35]: We should either make this no longer TBD or remove it.

File: handbook/ops/bizops/user_definitions.md
Comment (jyavorska) [inline handbook/ops/bizops/user_definitions.md:9]:
```suggestion
Users can be in a number of different states. We track the following categories in pings for each month. The descriptions below are framed as though we're talking about MAU, but the same definitions apply to whatever period is being considered.
```

File: handbook/product/index.md
Comment (jyavorska) [inline handbook/product/index.md:27]: Is everyone comfortable with explicitly setting this goal? Are there other components to the goal, or additional context to record?

File: handbook/ops/bizops/user_definitions.md
Comment (serinadawn) [reply handbook/ops/bizops/user_definitions.md:3]: Given the way you present the information in this pull request, it seems correct to no distinguish between on-premises in cloud.

File: handbook/ops/bizops/user_definitions.md
Comment (serinadawn) [inline handbook/ops/bizops/user_definitions.md:9]: I would remove the words "in pings" from the following sentence:" We track the following categories in pings for each month."

File: handbook/ops/bizops/user_definitions.md
Comment (serinadawn) [inline handbook/ops/bizops/user_definitions.md:23]: In this section to accommodate that we are describing metrics for both on-premises and cloud solutions could we replace the word "site" with "product" or something that does not infer a "location"?  Reach out if this feedback is unclear.

File: handbook/ops/bizops/user_definitions.md
Comment (serinadawn) [reply handbook/ops/bizops/user_definitions.md:34]: @jyavorska to get this PR merged and published let's just remove all the places where there is TBD.  We can go back in and add info once we have it for each of these product areas.

File: handbook/product/index.md
Comment (christinaforney) [reply handbook/product/index.md:27]: We can instead point at the OKR pages once they are updated rather than embedding the goal here.

File: handbook/ops/bizops/user_definitions.md
Comment (jyavorska) [reply handbook/ops/bizops/user_definitions.md:9]:
```suggestion
Users can be in a number of different states. We track the following categories for each month. The descriptions below are framed as though we're talking about MAU, but the same definitions apply to whatever period is being considered.
```

File: handbook/ops/bizops/user_definitions.md
Comment (jyavorska) [reply handbook/ops/bizops/user_definitions.md:23]:
```suggestion
A visitor is a unique person who showed up on the site and did anything (or nothing at all, apart from viewing the page.) This is important to measure because each of these users is potentially an active user, but it's also important that we don't count anyone who uses the product as an active user so that we can measure success of our features that are intended to convert them.
```

File: handbook/ops/bizops/user_definitions.md
Comment (jyavorska) [reply handbook/ops/bizops/user_definitions.md:34]: Yeah good call.

Comment (jyavorska) [reply handbook/ops/bizops/user_definitions.md:34]:
```suggestion
```

File: handbook/ops/bizops/user_definitions.md
Comment (jyavorska) [reply handbook/ops/bizops/user_definitions.md:35]:
```suggestion
```

File: handbook/ops/bizops/user_definitions.md
Comment (jyavorska) [reply handbook/ops/bizops/user_definitions.md:33]:
```suggestion
```

File: handbook/product/index.md
Comment (jyavorska) [reply handbook/product/index.md:27]: That makes sense. I'll keep this discussion open until we make a decision, then once it's in the OKR and linked to from here I'll resolve this thread.

File: handbook/ops/bizops/user_definitions.md
Comment (jyavorska) [reply handbook/ops/bizops/user_definitions.md:37]: @anna-mmikhova was the one tagged in the discussion, so if anyone has any ideas of additional items to add then please shout, otherwise I'll remove this section before merging and we can always follow up with additional improvements later.

File: handbook/ops/bizops/user_definitions.md
Comment (jyavorska) [reply handbook/ops/bizops/user_definitions.md:38]: @ebrodymoore you were tagged explicitly in the conversation in the doc, so tagging you here, but of course everyone is welcome to comment. Is there anything to add here, or can/should this come in an later iteration?

**Comment by jyavorska** [comment]
@anna-mmikhova @serinadawn @beyang @kelsey-brown @bcreager @christinaforney @rrhyne @dadlerj @malomarrec tagging you all in here as you were viewers in the original document; I'll also be sharing in the #product-led channel.

What I did here was try to convert the product led metrics refinement doc at https://docs.google.com/document/d/1o0dLmdRRI6uWIuAg_8VQw25KnTM1CBDKKR2K91SxpAI/edit?skip_itp2_check=true&pli=1# into a concrete PR that we can discuss as needed and merge. I think we're close to a first iteration thanks to all the discussions that had happened previously, but now is the chance to look at it in situ and see if it makes sense to go forward. It's intended to be a clear first iteration that we can then improve on as needed, but at least starts us from a clear, agreed upon starting point.

There are a list of next steps in the document at https://docs.google.com/document/d/1o0dLmdRRI6uWIuAg_8VQw25KnTM1CBDKKR2K91SxpAI/edit?skip_itp2_check=true&pli=1# which still need to happen after merging this. It would be helpful to also take a look and see if anything is missing.

**Comment by jyavorska** [comment]
My proposal for the next iteration of this would be for each product/engineering team to look at their area and ensure that their flow is covered for the definition of an engaged user. If not, add additional metrics for each.

We could consider tracking these as team-level metrics, but that can be a separate decision from just making sure we have a complete set of activities that represent an engaged user.

**Comment by ryphil** [comment]
@jyavorska - Do you plan to add a section on what it means to be a retained user? Our conversations on cloud are heading towards that being the top level metric as we need to get users into a retained state to ensure long-term success of Sourcegraph.

File: handbook/ops/bizops/user_definitions.md
Comment (jyavorska) [reply handbook/ops/bizops/user_definitions.md:37]: Removing since nobody had any ideas.

Comment (jyavorska) [reply handbook/ops/bizops/user_definitions.md:37]:
```suggestion
```

File: handbook/ops/bizops/user_definitions.md
Comment (jyavorska) [reply handbook/ops/bizops/user_definitions.md:37]: Removing since nobody had any ideas

Comment (jyavorska) [reply handbook/ops/bizops/user_definitions.md:37]:
```suggestion
```

File: handbook/ops/bizops/user_definitions.md
Comment (jyavorska) [reply handbook/ops/bizops/user_definitions.md:3]: Resolving this thread since we seemed to agree.

**Comment by jyavorska** [comment]
> @jyavorska - Do you plan to add a section on what it means to be a retained user? Our conversations on cloud are heading towards that being the top level metric as we need to get users into a retained state to ensure long-term success of Sourcegraph.

@ryphil probably not as part of this first iteration - in my conversations with @anna-mmikhova it sounded like this was getting stuck in part because we were trying to solve all the problems at once, and so we weren't making good progress on any single part of it. I added that as an explicit step to the next iteration, though, and am open to hearing if this whole PR should block on that definition.

File: handbook/ops/bizops/user_definitions.md
Comment (ebrodymoore) [inline handbook/ops/bizops/user_definitions.md:32]:
```suggestion
An active user is differentiated from a visitor by performing a qualifying event, which are intended to represent someone who has actively **engaged with** and **received value** from the product; something that takes a user from their first Search to establishing a habit around the core value proposition of the app. Qualifying events are [listed as 'Active' in Amplitude](https://analytics.amplitude.com/sourcegraph/govern/project/333976/events?filter=live)
```

**Review by ebrodymoore — APPROVED** [review]

File: handbook/ops/bizops/user_definitions.md
Comment (jyavorska) [reply handbook/ops/bizops/user_definitions.md]: Makes sense @ebrodymoore - for these, which are applicable to which? I'm happy to make the markdown change but am not sure what the answer is.

Comment (ebrodymoore) [reply handbook/ops/bizops/user_definitions.md]: Made the commit [here]!(https://github.com/sourcegraph/about/pull/4521/commits/03b83b9c65b799f641061bb62ea32a292d8ae182)

Comment (jyavorska) [reply handbook/ops/bizops/user_definitions.md]: Thanks so much @ebrodymoore!

File: handbook/product/index.md
Comment (jyavorska) [reply handbook/product/index.md]: @christinaforney I believe this is the last item for this first iteration - two questions:

- In terms of goal setting, do you already have something in mind for updating the OKRs?
- Are you good to go with the rest of the changes in this PR?

Thanks!

**Comment by jyavorska** [comment]
Moved to https://github.com/sourcegraph/handbook/pull/78 after handbook refactor.

**Pull request closed without merging**
